app/views/accounts/_email.html.slim

```diff
@@ -1,13 +0,0 @@
-.clearfix.border-top.border-blue-light
```

Comment:
This template is a copy of the webauthn template that is not used. I cleaned it up.